ui/StatusQ/src/StatusQ/Core/StatusScrollView.qml

```diff
@@ -28,42 +32,51 @@ import StatusQ.Core.Utils 0.1
 
    For a list of components available see StatusQ.
 */
-Flickable {
+
+T.ScrollView {
     id: root
 
-    // NOTE: this should be replaced with margins since Qt 5.15
-    property int padding: 8
-    property int topPadding: padding
-    property int bottomPadding: padding
-    property int leftPadding: padding
-    property int rightPadding: padding
+    readonly property Flickable flickable: root.contentItem
+
+    function ensureVisible(rect) {
+        Utils.ensureVisible(flickable, rect)
+    }
+
+    function applyFlickableFix() {
```

Comment:
Does it need to be public?

Reply:
Well I just left it public for any further needs. Because we only set it in `Component.onCompleted`, I wasn't sure that this will be enough in all other places, which I only checked briefly.

Setting it in function is the only possible way I could achieve. Binding would be much better and straightforward, but doesn't work for the object property 😑